SwiftUI/GraphicsHub/GraphicsHub/Catalogue/CatalogueItem.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  CatalogueItem.swift
+//  GraphicsHub
+//
+//  Created by Noah Pikielny on 9/7/21.
+//
+
+import SwiftUI
+
+struct CatalogueItem: View {
+    
+    var renderer: RendererCatalog.RenderingOption
+    
+    var body: some View {
+        VStack {
+            HStack {
+                VStack(alignment: .leading) {
+                    Text(renderer.name)
+                        .font(.title)
+                    Text(renderer.description)
+                        .font(.caption)
+                }
+                .multilineTextAlignment(.leading)
+                ScrollView {
+                    VStack(spacing: 0) {
+                        ForEach (0..<renderer.images.count) { index in
+                            let image = renderer.images[index]
+                            image
+                                .renderingMode(.original)
+                                .resizable()
+                                .frame(width: 155, height: 155)
+                                .cornerRadius(5)
+                        }
+                    }
+                }.frame(height: 200)
```

Comment:
put on the next line? not sure what the convention is, but on line 23 you put it on the next line